doc/maintenance.md

```diff
@@ -19,14 +17,12 @@ communication tools we use.
 
 - Added to the [pygmt-contributors team](https://github.com/orgs/GenericMappingTools/teams/pygmt-contributors) (gives 'write' permission to the repository)
 - Added as a collaborator on [DAGsHub](https://dagshub.com/GenericMappingTools/pygmt/settings/collaboration) (gives 'write' permission to dvc remote storage)
-- Added to the [PyGMT devs Slack channel](https://pygmtdevs.slack.com) (for casual conversations)
```

Comment:
I'd like to know if we should sunset the PyGMT devs Slack channel since it's no longer actively used and messages older than 90 days are deleted. We can use GitHub Discussions (not enabled for the PyGMT repository, but see https://github.com/GenericMappingTools/gmt/discussions for an example) for open discussions and emails for confidential discussions. 

@GenericMappingTools/pygmt-maintainers Please vote on this comment 👍 or 👎. Or leave your comments.

Reply:
In favour because we haven't used Slack much and their policy deletion policy is getting worse https://forums.slackcommunity.com/s/question/0D5Hq00009rcK9vKAE/free-workspace-one-year-deletion-policy?language=en_US. We could always send the Discourse forum message, or email if necessary.

Reply:
I agree on this. Additional to these aspects, I still face a strange behavior of getting notifications by Slack.